### DISCUSSION TITLE
Retrieving Templates Used on Subsites

### DISCUSSION THREAD

**Discussion by YTtatata** [pnp/powershell#3573] [Q&A]
I would appreciate the assistance of anyone knowledgeable about this matter.
I am currently participating in an SPO to SPO migration project,
I am using PnP.Powershell to investigate the status of the site collection from which we are migrating.

I am using the following command in PnP.Powershell to get the results,
I am having trouble getting the same results for the modern and classic team sites.

```
$siteUrl = "https://test.sharepoint.com/sites/siteCollectionName"
$con = Connect-PnPOnline $siteUrl -Interactive
$subSitesInfo = Get-PnPSubWeb -Recurse -Includes WebTemplate -Connection $con
$subSiteInfo.WebTemplate
```
Results→STS

It would be most desirable to be able to obtain values such as "STS#3" or "STS#0" as in the case of the following command,
However, we would like to know if there is any other way to determine the value.

Sample:
```
$siteUrl = "https://test.sharepoint.com/sites/siteCollectionName"
$con = Connect-PnPOnline $Data.url -Interactive
$siteCollectionInfo = Get-PnPTenantSite -Identity $siteUrl -Connection $con
$siteCollectionInfo.Template
```

**Comment by AndersRask** [comment]
Not sure why you see this.
If i do the same i get both template name and id

```
get-pnptenantsite | select template | group template | select name

Name
----
APPCATALOG#0
CONTENTCTR#0
GROUP#0
OFFILE#1
POINTPUBLISHINGHUB#0
POINTPUBLISHINGTOPIC#0
SitePagePublishing#0
SPSMSITEHOST#0
SRCHCEN#0
STS#3
TEAMCHANNEL#1
```
